python/ray/_private/services.py

```diff
@@ -1367,6 +1368,8 @@ def start_raylet(redis_address,
              to.
         worker_path (str): The path of the Python file that new worker
             processes will execute.
+        setup_worker_path (str): The path of the Python file that will set up
```

Comment:
Should this be a file path or a python module to import? The latter makes more sense to me (especially for library devs)

Reply:
I see, I hadn't thought about library devs customizing the setup_worker but that seems useful